src/hotspot/share/runtime/safepoint.cpp

```diff
@@ -294,7 +294,6 @@ void SafepointSynchronize::begin() {
             cur_state->examine_state_of_thread();
             if (!cur_state->is_running()) {
               still_running--;
-              Universe::heap()->report_java_thread_yield(cur);
```

Comment:
Is this line added by us? Otherwise, this is shared code for all the OpenJDK GCs. I don't feel we should remove this. What we can do is to keep the `report_java_thread_yield` implementation empty, and have another method that we use in the binding.

Reply:
This line, as well as the `report_java_thread_yield` method, are introduced in the following commit:

https://github.com/mmtk/openjdk/commit/5bc4501c2d299df388070b2277f8d23b26f1e51a

In the associated PR I created in `mmtk-openjdk` (https://github.com/mmtk/mmtk-openjdk/pull/108), the `report_java_thread_yield` function is called in `mmtk_stop_all_mutators` (in `openjdk/mmtkUpcalls.cpp`) after all threads come to a stop.